website/server/models/user/methods.js

```diff
@@ -20,10 +20,11 @@ schema.methods.getGroups = function getUserGroups () {
   return userGroups;
 };
 
-schema.methods.sendMessage = async function sendMessage (userToReceiveMessage, message) {
+schema.methods.sendMessage = async function sendMessage (userToReceiveMessage, translatedMessage, message) {
```

Comment:
I think this should be a two argument method. The first argument being the user and the second to be an options object with two properties: receiverMsg and senderMsg. If senderMsg is undefined, set it to receiverMsg. 

```is
let senderMsg = options.senderMsg || options.receiverMsg;
```